### ISSUE TITLE
Missing my module 'org'

### ISSUE THREAD

**Issue by xavierzhao** [Digital-Sapphire/PyUpdater#4]
I am using `PyUpdater` to build my app, but an error occurred when running my app.

here is build log:
`pyupdater build -F --app-version 1.0 main.py`

```
[INFO] wrote D:\sources\python\gc\.pyupdater\spec\win.spec
[INFO] PyInstaller: 3.1.1
[INFO] Python: 3.4.3
[INFO] Platform: Windows-8-6.2.9200
[INFO] UPX is not available.
[INFO] Extending PYTHONPATH with paths
['D:\\sources\\python', 'D:\\sources\\python\\gc\\.pyupdater\\spec']
[INFO] checking Analysis
[INFO] Building Analysis because out00-Analysis.toc is non existent
[INFO] Initializing module dependency graph...
[INFO] Initializing module graph hooks...
[INFO] Analyzing base_library.zip ...
[INFO] Processing pre-find module path hook   distutils
[INFO] running Analysis out00-Analysis.toc
[INFO] Analyzing D:\sources\python\gc\main.py
[INFO] Processing pre-safe import module hook   six.moves
[INFO] Processing pre-safe import module hook   win32com
[INFO] Processing pre-find module path hook   site
[INFO] site: retargeting to fake-dir 'd:\\python34\\lib\\site-packages\\PyInstaller\\fake-modules'
[INFO] Looking for import hooks ...
[INFO] Processing hook   hook-pkg_resources.py
[INFO] Processing hook   hook-sysconfig.py
[INFO] Processing hook   hook-cryptography.py
[INFO] Processing hook   hook-pydoc.py
[INFO] Processing hook   hook-PyQt5.py
[INFO] Processing hook   hook-distutils.py
[INFO] Processing hook   hook-pythoncom.py
[INFO] Processing hook   hook-xml.dom.domreg.py
[INFO] Processing hook   hook-win32com.py
[INFO] Processing hook   hook-certifi.py
[INFO] Processing hook   hook-encodings.py
[INFO] Processing hook   hook-jms_utils.py
[INFO] Processing hook   hook-xml.py
[INFO] Processing hook   hook-PyQt5.QtCore.py
[INFO] Processing hook   hook-PyQt5.QtWidgets.py
[INFO] Processing hook   hook-PyQt5.Qt.py
[INFO] Processing hook   hook-pywintypes.py
[INFO] Processing hook   hook-PyQt5.QtGui.py
[INFO] Processing hook   hook-requests.py
[INFO] Looking for ctypes DLLs
[INFO] Analyzing run-time hooks ...
[INFO] Including run-time hook 'pyi_rth_win32comgenpy.py'
[INFO] Including run-time hook 'pyi_rth_pkgres.py'
[INFO] Including run-time hook 'pyi_rth_qt5.py'
[INFO] Including run-time hook 'pyi_rth_qt5plugins.py'
[INFO] Looking for dynamic libraries
[WARNING] Attempted to add Python module twice with different upper/lowercases: PyQt5.Qt
[WARNING] Attempted to add Python module twice with different upper/lowercases: PyQt5.QtPrintSupport
[WARNING] Attempted to add Python module twice with different upper/lowercases: PyQt5.QtGui
[WARNING] Attempted to add Python module twice with different upper/lowercases: PyQt5.QtWidgets
[WARNING] Attempted to add Python module twice with different upper/lowercases: PyQt5.QtCore
[INFO] Looking for eggs
[INFO] Using Python library C:\WINDOWS\system32\python34.dll
[INFO] Found binding redirects:
[]
[INFO] Warnings written to D:\sources\python\gc\.pyupdater\work\win\warnwin.txt
[INFO] checking PYZ
[INFO] Building PYZ because out00-PYZ.toc is non existent
[INFO] Building PYZ (ZlibArchive) D:\sources\python\gc\.pyupdater\work\win\out00-PYZ.pyz
[INFO] checking PKG
[INFO] Building PKG because out00-PKG.toc is non existent
[INFO] Building PKG (CArchive) out00-PKG.pkg
[INFO] Bootloader d:\python34\lib\site-packages\PyInstaller\bootloader\Windows-64bit\run.exe
[INFO] checking EXE
[INFO] Building EXE because out00-EXE.toc is non existent
[INFO] Building EXE from out00-EXE.toc
[INFO] Appending archive to EXE D:\sources\python\gc\pyu-data\new\win.exe
[INFO] gc-win-1.0.zip has been placed in your new folder

[INFO] Build finished in 184.52 seconds.
```

and error log:
`gc.exe`

```
Traceback (most recent call last):
  File "<string>", line 10, in <module>
ImportError: No module named 'org'
main returned -1
```

`org` is my custom module

thanks for you help

**Comment by JMSwag** [comment]
@xavierzhao Please build your app with pyinstaller to see if you get the same results.

`$ pyinstaller -F main.py`

**Comment by xavierzhao** [comment]
@JMSwag not same results, my app is normally stated after `pyinstaller -F main.py`

**Comment by xavierzhao** [comment]
here is build log:

```
D:\sources\python\gc>pyinstaller -F main.py
72 INFO: PyInstaller: 3.1.1
73 INFO: Python: 3.4.3
73 INFO: Platform: Windows-8-6.2.9200
75 INFO: wrote D:\sources\python\gc\main.spec
77 INFO: UPX is not available.
79 INFO: Extending PYTHONPATH with paths
['D:\\sources\\python', 'D:\\sources\\python\\gc']
80 INFO: checking Analysis
88 INFO: Building because D:\sources\python\gc\main.py changed
88 INFO: Initializing module dependency graph...
92 INFO: Initializing module graph hooks...
95 INFO: Analyzing base_library.zip ...
2086 INFO: Processing pre-find module path hook   distutils
3298 INFO: running Analysis out00-Analysis.toc
3378 INFO: Analyzing D:\sources\python\gc\main.py
4199 INFO: Processing pre-safe import module hook   six.moves
4430 INFO: Processing pre-safe import module hook   win32com
5235 INFO: Processing pre-find module path hook   site
5237 INFO: site: retargeting to fake-dir 'd:\\python34\\lib\\site-packages\\PyInstaller\\fake-modules'
6539 INFO: Looking for import hooks ...
6548 INFO: Processing hook   hook-PyQt5.QtWidgets.py
6550 INFO: Processing hook   hook-pydoc.py
6552 INFO: Processing hook   hook-encodings.py
6569 INFO: Processing hook   hook-xml.etree.cElementTree.py
6571 INFO: Processing hook   hook-win32com.py
6927 INFO: Processing hook   hook-certifi.py
6930 INFO: Processing hook   hook-PyQt5.QtCore.py
7002 INFO: Processing hook   hook-PyQt5.py
7004 INFO: Processing hook   hook-PyQt5.QtGui.py
7428 INFO: Processing hook   hook-distutils.py
7430 INFO: Processing hook   hook-xml.py
7432 INFO: Processing hook   hook-pkg_resources.py
7435 INFO: Processing hook   hook-pythoncom.py
7662 INFO: Processing hook   hook-sysconfig.py
7665 INFO: Processing hook   hook-xml.dom.domreg.py
7667 INFO: Processing hook   hook-pywintypes.py
7887 INFO: Processing hook   hook-PyQt5.Qt.py
7931 INFO: Looking for ctypes DLLs
7947 INFO: Analyzing run-time hooks ...
7958 INFO: Including run-time hook 'pyi_rth_win32comgenpy.py'
7961 INFO: Including run-time hook 'pyi_rth_pkgres.py'
7963 INFO: Including run-time hook 'pyi_rth_qt5.py'
7965 INFO: Including run-time hook 'pyi_rth_qt5plugins.py'
7994 INFO: Looking for dynamic libraries
9261 WARNING: Attempted to add Python module twice with different upper/lowercases: PyQt5.Qt
9263 WARNING: Attempted to add Python module twice with different upper/lowercases: PyQt5.QtPrintSupport
9265 WARNING: Attempted to add Python module twice with different upper/lowercases: PyQt5.QtGui
9268 WARNING: Attempted to add Python module twice with different upper/lowercases: PyQt5.QtWidgets
9269 WARNING: Attempted to add Python module twice with different upper/lowercases: PyQt5.QtCore
9271 INFO: Looking for eggs
9272 INFO: Using Python library C:\WINDOWS\system32\python34.dll
9275 INFO: Found binding redirects:
[]
9303 INFO: Warnings written to D:\sources\python\gc\build\main\warnmain.txt
9377 INFO: checking PYZ
9384 INFO: Building because toc changed
9384 INFO: Building PYZ (ZlibArchive) D:\sources\python\gc\build\main\out00-PYZ.pyz
10008 INFO: checking PKG
10010 INFO: Building because toc changed
10011 INFO: Building PKG (CArchive) out00-PKG.pkg
26068 INFO: Bootloader d:\python34\lib\site-packages\PyInstaller\bootloader\Windows-64bit\run.exe
26071 INFO: checking EXE
26072 INFO: Building because out00-EXE.toc is bad
26072 INFO: Building EXE from out00-EXE.toc
26079 INFO: Appending archive to EXE D:\sources\python\gc\dist\main.exe
```

**Comment by JMSwag** [comment]
I think I see the problem. Would you mind posting the contents of your spec file?

D:\sources\python\gc.pyupdater\spec

**Comment by xavierzhao** [comment]
here is my spec:

```
# -*- mode: python -*-

block_cipher = None


a = Analysis(['D:\\sources\\python\\gc\\main.py'],
             pathex=['D:\\sources\\python\\gc\\.pyupdater\\spec'],
             binaries=None,
             datas=None,
             hiddenimports=[],
             hookspath=['d:\\python34\\lib\\site-packages\\pyupdater\\hooks'],
             runtime_hooks=[],
             excludes=[],
             win_no_prefer_redirects=False,
             win_private_assemblies=False,
             cipher=block_cipher)
pyz = PYZ(a.pure, a.zipped_data,
             cipher=block_cipher)
exe = EXE(pyz,
          a.scripts,
          a.binaries,
          a.zipfiles,
          a.datas,
          name='win',
          debug=False,
          strip=False,
          upx=True,
          console=True )
```

**Comment by JMSwag** [comment]
Update the analysis in your spec file like the example below & please reply with your results.

a = Analysis(['D:\sources\python\gc\main.py'],
             pathex=['D:\sources\python\gc\.pyupdater\spec','D:\sources\python\gc'],
             binaries=None,
             datas=None,
             hiddenimports=[],
             hookspath=['d:\python34\lib\site-packages\pyupdater\hooks'],
             runtime_hooks=[],
             excludes=[],
             win_no_prefer_redirects=False,
             win_private_assemblies=False,
             cipher=block_cipher)

**Comment by xavierzhao** [comment]
@JMSwag Thanks in advance, it works. I generated spec file using`pyupdater init` command.

**Comment by JMSwag** [comment]
@xavierzhao No problem. I'll reopen this issue for now until I implement a fix for the v2.0 release.

Please report any other issues you find.

**Comment by xavierzhao** [comment]
@JMSwag Is it possible not create a one-file bundled executable? My `gc.exe` is too big cause running slowly.

**Comment by JMSwag** [comment]
Closed in f1ed03c
